funnel/models/commentvote.py

```diff
@@ -64,10 +64,10 @@ def getvote(self, user):
 
 class Vote(BaseMixin, db.Model):
     __tablename__ = 'vote'
-    user_id = db.Column(db.Integer, db.ForeignKey('user.id'), nullable=False)
+    user_id = db.Column(None, db.ForeignKey('user.id'), nullable=False)
```

Comment:
What does `None` do here?

Reply:
It picks up the column type from the foreign key. Using `None` avoids errors where the types are mismatched, and so is part of our code style.